src/modules/75-cf/components/FlagActivation/FlagPrerequisites.tsx

```diff
@@ -395,58 +359,72 @@ export const FlagPrerequisites: React.FC<FlagPrerequisitesProps> = props => {
-                <RbacOptionsMenuButton
-                  items={[
-                    {
-                      icon: 'edit',
-                      text: getString('edit'),
-                      onClick: handlePrerequisiteInteraction('edit', elem),
-                      disabled: featureFlag.archived,
-                      permission: rbacPermission,
-                      ...planEnforcementProps
-                    },
-                    {
-                      icon: 'cross',
-                      text: getString('delete'),
-                      onClick: handlePrerequisiteInteraction('delete', elem),
-                      disabled: featureFlag.archived,
-                      permission: rbacPermission,
-                      ...planEnforcementProps
-                    }
-                  ]}
-                />
-              </Layout.Horizontal>
-            ))}
-          <RbacButton
-            data-testid="add-prerequisites-button"
-            minimal
-            intent="primary"
-            icon="small-plus"
-            text={getString('cf.shared.prerequisites')}
-            onClick={() => {
-              setEditingPrerequisites(false)
-              openModalPrerequisites()
-            }}
-            disabled={featureFlag.archived}
-            permission={rbacPermission}
-            {...planEnforcementProps}
-          />
-        </Layout.Vertical>
-      </Collapse>
-    </Container>
+    <Layout.Vertical margin={{ bottom: 'medium', top: 'medium' }} id="flagPrereqsContainer">
+      <Layout.Vertical margin={{ bottom: 'medium' }}>
+        <Text className={css.subheading}>
+          <StringWithTooltip stringId="cf.shared.prerequisites" tooltipId="ff_ffPrerequisites_heading" />
+        </Text>
+        <Text className={css.subheadingHelptext}>{getString('cf.featureFlags.prerequisitesDesc')}</Text>
+      </Layout.Vertical>
+      {Boolean(featureFlag.prerequisites?.length) && (
+        <Layout.Horizontal flex={{ justifyContent: 'flex-start' }} className={css.prereqHeaders}>
+          <Text width="45%">{getString('flag')}</Text>
+          <Text width="45%">{getString('cf.shared.variation')}</Text>
+        </Layout.Horizontal>
+      )}
+      {Boolean(featureFlag.prerequisites?.length) &&
+        featureFlag.prerequisites?.map((elem, i) => {
+          const isFlagTypeBoolean = ['true', 'false'].includes(elem.variations[0])
+          const flagVariation = elem.variations[0]
+          return (
+            <Layout.Horizontal key={i} flex={{ alignItems: 'center' }}>
+              <Text width="45%" lineClamp={1} className={css.flagDetails}>
+                <VariationTypeIcon style={{ transform: 'translateY(1px)' }} multivariate={!isFlagTypeBoolean} />
+                {elem.feature}
+              </Text>
+              <Text width="45%" className={css.flagDetails}>
+                {!isFlagTypeBoolean ? flagVariation : getString(`cf.featureFlags.${flagVariation}` as StringKeys)}
```

Comment:
String concatenation like this prevents the string tools from working. Can we pull this out into a mapping function or (since it's just true/false), use a ternary to put the strings in whole?
```suggestion
                {!isFlagTypeBoolean ? flagVariation : getString(flagVariation === 'true' ? 'cf.featureFlags.true' : 'cf.featureFlags.false')}
```